home-data-migration/src/main/resources/db/changelog/0.0.1/Issue96.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="Issue96.001" author="Abhai2016">
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="account">
+            <column name="id" value="1"/>
+            <column name="create_date" value="2021-01-13 11:30:00"/>
+            <column name="email"  value="admin@example.com"/>
+            <column name="password" value="{bcrypt}$2a$10$dp.3Muw2krCykzwILSlhse16CdowCELLuxwyeiB1OFYjeuYbRaqbC"/>
+            <column name="first_name" value="User"/>
+            <column name="last_name" value="User"/>
+            <column name="expired" value="false"/>
+            <column name="locked" value="false"/>
+            <column name="credentials_expired" value="false"/>
+            <column name="enabled" value="true"/>
+            <column name="update_date" value="2021-01-13 11:30:00"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue96.002" author="Abhai2016">
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="account">
+            <column name="id" value="2"/>
```

Comment:
We do not need a user added as liquibase migration. Admin will be enough. Please remove a user.

To be clear we can create any user if we have a user in admin role.

Reply:
Done.